content/en/docs/concepts/policy/_index.md

```diff
@@ -1,11 +1,68 @@
 ---
 title: "Policies"
 weight: 90
+no_list: true
 description: >
-  Policies you can configure that apply to groups of resources.
+  Manage security and best-practices with policies.
 ---
 
-{{< note >}}
-See [Network Policies](/docs/concepts/services-networking/network-policies/)
-for documentation about NetworkPolicy in Kubernetes.
-{{< /note >}}
+<!-- overview -->
+
+Kubernetes policies are configurations that manage other configurations or runtime behaviors. Kubernetes offers various forms of policies, described below:
+
+<!-- body -->
+
+## Apply policies using API objects
+
+ Some API objects act as policies. Here are some examples:
+* [NetworkPolicies](/docs/concepts/services-networking/network-policies/) can be used to restrict ingress and egress traffic for a workload.
+ * [LimitRanges](/docs/concepts/policy/limit-range/) manage resource allocation constraints across different object kinds.
+ * [ResourceQuotas](/docs/concepts/policy/resource-quotas/) limit resource consumption for a {{< glossary_tooltip text="namespace" term_id="namespace" >}}.
+
```

Comment:
:thought_balloon: we could link to `#apply-policies-using-validatingadmissionpolicy` here and explain why it has its own heading.

Reply:
Since the #apply-policies-using-validatingadmissionpolicy itself links to `/docs/reference/access-authn-authz/validating-admission-policy/` should we consider combining it with this section instead?